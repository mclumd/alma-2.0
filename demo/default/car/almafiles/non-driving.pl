fif(and(rel(electric_car, X), not(rel(charged, X))), not(rel(powered, X))).

obs(quote(rel(gas_tank_car, ford))).
obs(quote(not(rel(fueled, ford)))).

obs(quote(rel(electric_car, prius))).
obs(quote(rel(gas_tank_car, prius))).
obs(quote(not(rel(charged, prius)))).
obs(quote(not(rel(fueled, prius)))).

obs(quote(rel(electric_car, tesla))).
obs(quote(not(rel(charged, tesla)))).
