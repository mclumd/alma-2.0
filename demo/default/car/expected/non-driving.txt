rel(car, ford)
rel(gas_tank_car, ford)
~rel(powered, ford)
~rel(fueled, ford)
abnormal(ford, car, "~rel(powered, ford)")
rel(car, prius)
rel(gas_tank_car, prius)
rel(electric_car, prius)
~rel(charged, prius)
~rel(fueled, prius)
~rel(powered, prius)
abnormal(prius, car, "~rel(powered, prius)")
rel(car, tesla)
rel(electric_car, tesla)
~rel(charged, tesla)
~rel(powered, tesla)
abnormal(tesla, car, "~rel(powered, tesla)")
